torchvision/datasets/_stereo_matching.py

```diff
@@ -120,7 +127,7 @@ def __getitem__(self, index: int) -> Tuple:
             ) = self.transforms(imgs, dsp_maps, valid_masks)
 
         if self._has_built_in_disparity_mask or valid_masks[0] is not None:
-            return imgs[0], imgs[1], dsp_maps[0], valid_masks[0]
+            return imgs[0], imgs[1], dsp_maps[0], valid_masks[0]  # type: ignore[return-value]
```

Comment:
Why do we need that ignore?

Reply:
mypy doesn't pick up the `valid_masks[0] is not None` condition unfortunately :/

Reply:
I don't know the exact error message, but it is probably rather complaining that we annotate this with `np.ndarray` and the output of `self.transforms` is `Any`. Could you check again?

Reply:
Actually it's complaining about the size of the returned tuple on my end :/

Reply:
Solved it by casting `valid_masks[0]` to its actual type